apps/extension/e2e/pom/onboarding.ts

```diff
@@ -1,3 +1,4 @@
+import { devnetWallet } from '../fixtures'
```

Comment:
`devnetWallet` -> `berkeleyWallet`? It would be great if we could eventually make these network agnostic!